packages/framework/tinylicious-client/package.json

```diff
@@ -79,7 +79,7 @@
 		"typescript": "~4.5.5"
 	},
 	"peerDependencies": {
-		"fluid-framework": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0"
+		"fluid-framework": "workspace:~"
 	},
```

Comment:
I think the safest thing is actually to remove the peer dep altogether. The build tools don't  handle the field so we can't count on them doing the right thing. Best to remove this while we figure it out. Same for azure-client I think.